index.js

```diff
@@ -20,6 +20,6 @@ module.exports = function (func, cb) {
   }, Array.prototype.slice.call(arguments, 2) );
 
   if (!async) {
-    cb(answer);
+    process.nextTick(cb.bind(null, answer));
```

Comment:
Oh wow. That happened a long time ago and I forgot. Where else have I screwed that up?